gitwrapperlib/gitwrapperlib.py

```diff
@@ -159,6 +165,25 @@ def get_current_branch(self):
                      if branch.startswith('*')),
                     None)
 
+    def get_default_branch(self):
+        """Returns the remote default branch."""
+        branch = None
+
+        try:
+            remote = str(self._git.remote('show', 'origin'))
+            match = re.search(r'HEAD branch: (\S+)', remote)
+            if match:
+                branch = match.group(1)
+                if branch == '(unknown)':
+                    branch = None
```

Comment:
We can raise the exception here, there is no benefit of the conditional later.

Reply:
The conditional handles both the no match case and the '(unknown)' case.

Reply:
What i mean is that we do not need to assign branch to None and raise later if branch is None, we can raise the exception right on line 178